uint32 frameNumber
byte markerNumber
bool occluded
float64 x_mm
float64 y_mm
float64 z_mm
